src/main/java/echo/task/ToDos.java

```diff
@@ -0,0 +1,40 @@
+package echo.task;
+
+/**
+ * Represents a to-do task with descriptions
+ *
+ * @author Ernest Lim
+ */
+public class ToDos extends Task {
```

Comment:
You can consider renaming the `ToDos` class (and every other instance of it) to be singular as it represents only 1 task. It would then adhere to the [coding standard](https://se-education.org/guides/conventions/java/index.html#naming), which states that:
- Plural form should be used on names representing a collection of objects.